core/src/main/scala/org/ensime/indexer/SearchService.scala

```diff
@@ -220,8 +220,8 @@ class SearchService(
     backlogActor ! FileUpdate(f, symbols)
   }(workerEC)
 
-  def shutdown(): Unit = {
-    db.shutdown()
+  def shutdown()(implicit ec: ExecutionContext): Unit = {
+    db.shutdown
```

Comment:
also why is workerEC passed in explicitly everywhere, shouldn't it be taken implicitly just right?